plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationPuckManager.kt

```diff
@@ -93,6 +93,7 @@ internal class LocationPuckManager(
     }
     delegateProvider.getStyle {
       initialize(it)
+      locationLayerRenderer.showBearingImage(settings.showBearingImage)
```

Comment:
looks a bit strange - can it be moved inside `initialize` function above?

Reply:
That's right, will move it.